src/config_parse.c

```diff
@@ -192,7 +191,7 @@ static int parse_section_header(git_config_parser *reader, char **section_out)
 	do {
 		if (git__isspace(c)){
 			name[name_length] = '\0';
-			result = parse_section_header_ext(reader, line, name, section_out);
+			result = parse_section_header_ext(reader, line, pos, name, section_out);
```

Comment:
I really don't like this method name. It's not an extended version of the normal `parse_section_header`, but instead it parses the subsection-part of section headers (e.g. the quoted part). So I think it should just be called `parse_subsection_header` or something like that.

Reply:
Agreed.  Fixed.